article/sections/_4_graph_construction.tex

```diff
@@ -1,3 +1,90 @@
+
+To begin, we adopt the position of a researcher concerned about population level relationships.
+This means we will think through what is a likely generative model for all individuals.
+Later on, we will devote time to thinking about how subgroups and individual heterogeneity may affect our causal graphs.
+
+Now, we add our first variables to our graph, the outcome variable(s) of interest.
+Then, we note all the variables that we believes to influence the variable(s) of interest.
+We refer to these variables to as our initial explanatory variables.
+
+Next, we iterate through the initial explanatory variables.
+For each current explanatory variable in the iteration, we think of variables that may modify the effect of the current explanatory variable on the outcome(s) of interest.
+We refer to these variables as effect modifiers.
+Note that some effect modifiers may be a part of our initial explanatory variables.
+For any effect modifiers that one thinks of, outside of the initial explanatory variables, add them to our causal graph.
+
+After adding explanatory and effect modifying variables to the graph, we turn our attention to mediating variables.
+Again, we iterate through each explanatory variable.
+On each iteration, we brainstorm all the variables through which our current explanatory variable influences the outcome.
+For instance, consider how the presence of a bike lane influences bicycle mode choice.
+We hypothesize that an individual's subjective perception of safety is be the primary (or sole) variable through which bicycle lane presence influences mode choice.
+Accordingly, we add subjective perception of safety to our causal graph for travel mode choice.
+
+Coming to our second to last category of variables, we think of confounding variables.
+The process is similar to how we generated effect modifying variables.
+We iterate through each of the explanatory, mediating, and effect modifying variables, thinking specifically of any variables that both cause the current variable in the iteration and cause the outcome variable(s).
+We call these variables, which cause our outcome and current variables in the iteration, confounding variables.
+We will add each of them to our causal graph.
+
+For the last set of variables, we should explicitly consider the role of time, even in research that may be cross-sectional due to the data that is available to us.
+In reality, how do we think our system evolves over time?
+If we consider multiple observations of a given decision maker, how do variables observed of that decision maker at time $t$ partially cause future variables important to the context or outcomes for that decision maker at time $t+1$?
+How do the actions of a decision maker $i$ at time $t$ partially cause the future context or outcomes of a decision maker $j$?
+We should add explicit nodes to our graph, subscripted or denoted by time, to show the cross-time causal relationships in our system.
```

Comment:
Hmm, for line 45, doesn't the "future context" specify the reference to a later time? The idea that $i$'s actions cause $j$'s decision within the current time is most closely related to the area of network interference within causal inference.

Now that you mention it, I realize I've neglected any explicit mention of network interference (mostly because I haven't done any causal inference work on the topic and it isn't foremost in my mind).

Nevertheless, I still don't think this paragraph is the best place for that. Interference within the current time would show up as a node on one individual's graph that happens to be the outcome of another individual. I don't think there's anything special about that other individual's outcome node, save maybe an undirected edge to the current indivudal's outcome node. All of the special treatment happens in the analysis of data subject to network interference. Not in the graph construction.

I should add a reference to the topic, likely in the "catch-all" discussion of causal inference and transportation choice modelling that didn't make it into the body of this chapter.

Reply:
I think what I should've done is delete "future context" and keep my added suggestion there because it accounts for network interference and future context.